core/CfgFunctions.hpp

```diff
@@ -1,77 +1,74 @@
-class FW {
-	tag = "FW";
-
-	class Core {
-		file = "core\functions";
-		class addItemOrg {};
-		class addItemRandomOrg {};
-		class addItemVehicleOrg {};
-		class addItemVehicleRandomOrg {};
-		class addTeam {};
-		class alive {};
-		class areaCount {};
-		class canAttachItem {};
-		class canLinkItem {};
-		class casualtyCount {};
-		class casualtyPercentage {};
-		class checkClassname {};
-		class countTeam {};
-		class createRespawnMarker {};
-		class debugMessage {};
-		class endMission {};
-		class eventDisconnect {};
-		class eventKilled {};
-		class eventPlayerSpawned {};
-		class eventRespawned {};
-		class eventSpawned {};
-		class forceTerrainGrid {};
-		class getDamagedAssets {};
-		class getTeamVariable {};
-		class getEmptyPositions {};
-		class hasExtracted {};
-		class inVehicle {};
-		class makeUnitsList {};
-		class notTrackUnit {};
-		class randomRange {};
-		class registerModule {};
-		class removeAllGear {};
-		class removeAllVehicleGear {};
-		class setTeamVariable {};
-		class spectatePrep {};
-		class stackNames {};
-		class startingCount {};
-		class trackAsset {};
-		class trackUnit {};
-		class untrackUnit {};
-		class briefing {};
-		class gear {};
-		class vehGear {};
-		class postChecks {};
-		class gearExport {};
-	};
+class Core {
```

Comment:
Does this mean framework function calls will no longer be prepended with "FW"? If so, have all the relevant changes to function calls been made and tested?

Reply:
They are prefixed with Core, this change makes it so a module can add a function to the FW group by defining another function group. The core cfgFunctions has simply been made into a series of function group defines with the FW tag group being defined in the description

Reply:
Sounds good.